### DISCUSSION TITLE
What happens if Windows' power mode and G-helper's performance mode are different?

### DISCUSSION THREAD

**Discussion by cheonmux** [seerge/g-helper#1028] [Q&A]
Hi,

First as all, thank you so much for developing G-Helper.

The program works perfectly, and I have a question about the power mode of Windows.

I've been experiencing random high latency issues, and I've read a lot on reddit that it's related to the power mode of windows.

And i'm using my laptop mostly in Slient mode due to fan noise. 

What happens if I set it to Silent mode in G-Helper and then set the power mode in Windows to Better performance or Maximum Perfomance?

I wonder if it's okay to use this way

Thanks!

**Comment by IceStormNG** [comment]
There is a  setting in GHelper to prevent Ghelper from touching the Windows power plan. Then you can set them individually. The powerplans and the GHelper "performance mode" are not tied to each other.
By default. Ghelper changes the power plan according to your performance mode, but this is because it replicates what AC does. If you disable the setting, you can leave Windows at "Balanced" or "Best performance". Silent defaults to "Best Energysaving", which greatly caps the CPU performance and can cause high latency issues.
